.changeset/brown-singers-sit.md

```diff
@@ -0,0 +1,26 @@
+---
+'@openfn/language-dhis2': major
```

Comment:
We've also removed the discover function

Can you please check this PR really carefully and make sure this list is complete?